openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/EditTestCaseModal.tsx

```diff
@@ -62,6 +69,25 @@ const EditTestCaseModal: React.FC<EditTestCaseModalProps> = ({
     [testCase]
   );
 
+  const isComputeRowCountFieldVisible = useMemo(() => {
+    return selectedDefinition?.supportsRowLevelPassedFailed ?? false;
```

Comment:
```suggestion
    return Boolean(selectedDefinition?.supportsRowLevelPassedFailed);
```

Reply:
`selectedDefinition?.supportsRowLevelPassedFailed` itself is a boolean value, i am just giving fallback of default value.

Reply:
@ShaileshParmar11, to return a boolean value, we can use Boolean. If `selectedDefinition?.supportsRowLevelPassedFailed` is nullish, it will return false.